### DISCUSSION TITLE
RustDesk Silent Installation

### DISCUSSION THREAD

**Discussion by abdullah-erturk** [rustdesk/rustdesk#3481] [General]
https://github.com/abdullah-erturk/RustDesk-Silent-Installation

I prepared a small script for silent installation of Rustdesk software.

If you have your own Rustdesk Server, you must specify your server's ip/domain and key information in the script file. So after installation Rustdesk software will automatically connect to your server.

If you do not have a Rustdesk Server, you can install it without making any changes, in which case the Rustdesk software will be installed in its default settings.

**Comment by dannyx93** [comment]
Works after GPO for users without administrative rights?

**Reply by abdullah-erturk** [reply]
If program installation is blocked for users on company computers, permission must be granted by the group policy in order for this batch file to run.

**Reply by dannyx93** [reply]
the script is automatically blocked by antivirus

**Reply by abdullah-erturk** [reply]
The codes of the script file are open source, there are no malicious code lines...

You should whitelist it in your antivirus application.

**Reply by ghost** [reply]
Maybe you just wanna try adding "--silent-install" argument to your .exe installer, if you don't want to whitelist the script in your antivirus console

**Comment by redneck62** [comment]
Hello,
Is it possible to specify a permanent password automatically with your script ?
Merci

**Reply by abdullah-erturk** [reply]
If there was a password option in the command support parameters of the Rustdesk application, it would be what you said.
As far as I know, there is no password option in the command support parameters of the Rustdesk application.
